data/human/deep jobs.txt

```diff
@@ -403,27 +409,27 @@ mission "Secure Transport Job (Secret) [0]"
 	to offer
 		random < 25
 		not "deep security reveal"
-		"reputation: Deep Security" >= -50
+		"reputation: Deep Security" >= 0
+		"reputation: Deep Security" >= - "secure deep package" * 3
 	source
 		attributes "deep"
-		not attributes "station" "moon"
+		not attributes "station"
 	destination
+		distance 2 7
 		attributes "deep"
 		not planet "Valhalla"
-		not attributes "station" "moon"
+		not attributes "station"
 	on accept
 		dialog phrase "secure package pickup"
 	on visit
 		dialog phrase "generic cargo on visit"
 	on complete
-		payment 50000 10000
-		"reputation: Deep Security" += 0.5
+		payment 3000 750
+		"reputation: Deep Security" += 1
 		"secure deep package" ++
 		dialog phrase "secure package dropoff payment"
 	on fail
```

Comment:
Make this change to any mission with an `on fail`. I don't support aborting missions causing a negative reputation hit, and I have no idea why not only did the PR that added these jobs have that but also why it punished aborting a mission more than failing it, as failing a mission can mean outright losing the cargo or passengers do to a destroyed ship; I imagine the Deep would be more concerned with loss of cargo or personnel than you just leaving them at a spaceport somewhere.
```suggestion
	on abort
	on fail
```